trunk/src/app/srs_app_rtc_source.hpp

```diff
@@ -335,6 +342,13 @@ class SrsRtcFrameBuilder
     uint16_t header_sn_;
     uint16_t lost_sn_;
     int64_t rtp_key_frame_ts_;
+
+    // Audio jitter buffer, map sequence number to packet
+    std::map<uint16_t, SrsRtpPacket*> audio_buffer_;
```

Comment:
It seems that the audio buffer is similar to SrsRtpRingBuffer; please consider whether you can use it directly.